modules/contact/src/ContactPenetrationAuxAction.C

```diff
@@ -45,10 +45,7 @@ ContactPenetrationAuxAction::act()
 
   {
     InputParameters params = _factory.getValidParams("PenetrationAux");
-
-    MultiMooseEnum execute_options = SetupInterface::getExecuteOptions();
-    execute_options = "initial linear";
-    params.set<MultiMooseEnum>("execute_on") = execute_options;
+    params.set<ExecFlagEnum>("execute_on") = std::vector<ExecFlagType>({EXEC_INITIAL, EXEC_LINEAR});
```

Comment:
In general, I maintained the use of quite mode. For specific objects like this it probably is designed to only work with these flags (it should probably be suppressed after setting). But, for system level changes (i.e., Output) quiet mode should be used because we are just setting sensible defaults not trying to dictate proper behavior.